### PR TITLE
Runbook: Change EBS Volume to GP3 Type

### PR DESCRIPTION
**Pull request by shloka-bhalgat-unskript**
### Testing
Since there were no failing cases the remediation lego won't run

https://user-images.githubusercontent.com/110628398/231735563-1b65b145-2a5f-4216-b932-15865fe0eee7.mov

### Checklist:
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published. 

### Documentation
Make sure that you have documented corresponding changes in this repository.

### CODE REVIEW

File: AWS/Change_AWS_EBS_Volume_To_GP3_Type.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "name": "Change AWS EBS Volume To GP3 Type",
+    "description": "This runbook can be used to change the type of an EBS volume to GP3(General Purpose 3). GP3 type volume has a number of advantages over it's predecessors. gp3 volumes are ideal for a wide variety of applications that require high performance at low cost",
+    "uuid": "",
```
Comment (dougsillars) [inline AWS/Change_AWS_EBS_Volume_To_GP3_Type.json:4]: needs UUID?

**Review by dougsillars — CHANGES_REQUESTED** [review]

File: AWS/Change_AWS_EBS_Volume_To_GP3_Type.json (hunk 1 above)
Comment (shloka-bhalgat-unskript) [reply AWS/Change_AWS_EBS_Volume_To_GP3_Type.json]: Done

**Review by dougsillars — APPROVED** [review]

**Comment by amit-chandak-unskript** [comment]
@shloka-bhalgat-unskript this is a remediation runbook, right? that means the check corresponding to this runbook will return failed objects, which can be passed as param to this runbook. So having region as the runbook param doesnt look right.  Whats the check corresponding to this runbook?

**Review by amit-chandak-unskript — CHANGES_REQUESTED** [review]
Take care of comments

**Comment by shloka-bhalgat-unskript** [comment]
> @shloka-bhalgat-unskript this is a remediation runbook, right? that means the check corresponding to this runbook will return failed objects, which can be passed as param to this runbook. So having region as the runbook param doesnt look right. Whats the check corresponding to this runbook?

@amit-chandak-unskript region is an optional parameter. We had decided to have this as a part of the parameters because if the user intends to run the runbook by itself than through the healthcheck  feature, it needs to have region as an optional param. (so that the `list all aws regions` lego is triggered in the runbook)

**Comment by amit-chandak-unskript** [comment]
> > @shloka-bhalgat-unskript this is a remediation runbook, right? that means the check corresponding to this runbook will return failed objects, which can be passed as param to this runbook. So having region as the runbook param doesnt look right. Whats the check corresponding to this runbook?
> 
> @amit-chandak-unskript region is an optional parameter. We had decided to have this as a part of the parameters because if the user intends to run the runbook by itself than through the healthcheck feature, it needs to have region as an optional param. (so that the `list all aws regions` lego is triggered in the runbook)

@shloka-bhalgat-unskript i can understand region parameter, but how is the information from the failed objects from the check corresponding to this remediation runbook passed? It should have an optional parameter list of ebs volumes (volume id, region) tuple.

**Review by dougsillars — APPROVED** [review]

**Review by amit-chandak-unskript — APPROVED** [review]

**Pull request merged**